components/oscd-filter-button/src/oscd-filter-button.ts

```diff
@@ -31,13 +31,23 @@ import {
   }
   
   /**
-   * A mwc-list with mwc-textfield that filters the list items for given or separated terms
+   * @prop {string} header - The header of the dialog.
+   * @prop {string} icon - The icon of the button.
+   * @prop {boolean} disabled - If true, the button is disabled.
+   * @slot - The default slot.
+   * @example <oscd-filter-button></oscd-filter-button>
+   * @summary Displays a button that opens a dialog with a list filterable by given search terms
+   * @tagname oscd-filter-button
+   * @cssprop --mdc-theme-on-surface - The color of the icon.
    */
   export class OscdFilterButton extends OscdFilteredList {
+    /** the header of the dialog. */
```

Comment:
The texts above the properties aren't necessary. This actually does exactly the same thing as the annotation does. It's not wrong to keep this, the CEM analyzer wont create a double entry, but it serves no real purpose other then maybe a bit readability in the code? It's up to you if you want to keep it in 😄